src/utils/dmg.c

```diff
@@ -378,10 +387,16 @@ pool_op_hdlr(int argc, char *argv[])
 
 	case POOL_EXCLUDE:
 		/* Only support exclude single target XXX */
-		D_ASSERT(targets->rl_nr == 1);
+		D_ASSERT(ranks->rl_nr == 1);
 		tgt_list.tl_nr = 1;
-		tgt_list.tl_ranks = targets->rl_ranks;
-		tgt_list.tl_tgts = &tgt;
+		tgt_list.tl_ranks = ranks->rl_ranks;
+		if (targets != NULL) {
+			D_ASSERT(targets->rl_nr == 1);
+			tgt_list.tl_tgts = (int *)targets->rl_ranks;
+		} else {
+			tgt_list.tl_tgts = &tgt;
```

Comment:
Where are we setting the value for `tgt`?

Reply:
tgt = -1 in this case means all targets will on this rank will be excluded.